app/Http/Services/Contents/It/Event/ConnpassEventService.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace App\Http\Services\Contents\It\Event;
+
+/**
+ * ConnpassEventService.
+ *
+ * ConnpassEventを扱うService
+ */
+class ConnpassEventService
+{
+    /**
+     * Create a new instance.
+     */
+    public function __construct()
+    {
+    }
+    
+    /**
+     * ConnpassAPIにリクエストするURLの生成
+     * @param リクエストパラム
+     * @return string ConnpassAPIにリクエストするURL
+     */
+    private function _createRequestConnpassApiUrl($param) {
```

Comment:
__[code_sniffer]__
* Method name "_createRequestConnpassApiUrl" should not be prefixed with an underscore to indicate visibility
* Opening brace should be on a new line